Web/src/components/games/PandaSequence/GameDisplay.tsx

```diff
@@ -57,19 +71,29 @@ const Score = styled.h3`
   padding-top: 6px;
 `;
 
+/**
+ * This component handles both display and input mode.
+ * Display Mode: Element in sequence is animated according to "displaying" prop
+ * Input Mode: Element tapped is animated based on "selected" state
+ */
 const GameDisplay: React.FC<IProps> = ({
-  mode, secondsLeft, score, active, handleInputEvent,
+  mode, secondsLeft, score, processInput, timestep, displaying,
 }) => {
-  const onTap = (index:number) => {
-    // TODO: Animate tapped component to confirm user input
-    handleInputEvent(index);
-  };
-
   let balloons;
   if (mode === PandaSequenceMode.DISPLAY) {
-    balloons = Array.from(Array(5).keys()).map((i) => <Balloon key={i} style={{ width: active === i ? '100px' : '70px' }} />);
+    balloons = Array.from(Array(5).keys()).map((index) => (
```

Comment:
```suggestion
    balloons = Array(5).map((_, index) => (
```

Reply:
this doesn't work unless you fill the array with some dummy values lol.

_just javascript things_

Reply:
Ah I wanted to do it this way too! Unfortunately it doesn't seem to work because `Array(5)` creates an array of empty objects, which end up not being mapped to anything 😥

Reply:
Oof. TIL

Reply:
a slightly cleaner but still disgusting alternative is:
`Array(5).fill(null).map((_, idx) => idx)`

Reply:
👍 Will switch over!

Reply:
Ok I actually think I have no choice but to use my original syntax. The original is creating an array of `[0,1,2,3,4]` and then mapping those to Balloons with keys set as the list value.

`Array(5).fill(null).map((_, idx) => idx)` is using the list index as the Balloon key, which throws an eslint warning given [this](https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys).

I considered using `[...Array(5).keys()].map((idx) => ...` but it seems like we need to enable this feature in the tsconfig with the `--downlevelIteration` flag. Might as well keep the original syntax at this point 😂

Reply:
Well, if the number of balloons is always constant, I don't see why we shouldn't just do a `// eslint-disable-next-line` the index as key thing.

I always felt that was a bit of a dumb heuristic, similar to the react-hooks/exhaustive-deps

Reply:
Implementing this suggestion!